### PR TITLE
Sub functions

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs [hunk 1]
```diff
@@ -15,5 +18,21 @@ public static ProgressDialog CreateDialog(string title, string message, Context
...
+            var preferencesEditor = PreferenceManager.GetDefaultSharedPreferences(Application.Context).Edit();
+            new AlertDialog.Builder(context)
+                .SetCancelable(false)
+                .SetTitle("Are you sure to logout?")
+                .SetPositiveButton("Yes", (dce, e) =>
+                {
+                    preferencesEditor.PutString("UserId", string.Empty);
+                    preferencesEditor.PutBoolean("IsLogged", false);
+                    preferencesEditor.Commit();
+                    context.StartActivity(typeof(LoginActivity));
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs]: Finish the current activity

File: ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs [hunk 2]
```diff
@@ -15,5 +18,21 @@ public static ProgressDialog CreateDialog(string title, string message, Context
...
+        public static void ShowLogoutDialog(Context context)
+        {
+            var preferencesEditor = PreferenceManager.GetDefaultSharedPreferences(Application.Context).Edit();
+            new AlertDialog.Builder(context)
+                .SetCancelable(false)
+                .SetTitle("Are you sure to logout?")
+                .SetPositiveButton("Yes", (dce, e) =>
+                {
+                    preferencesEditor.PutString("UserId", string.Empty);
+                    preferencesEditor.PutBoolean("IsLogged", false);
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs]: Put the string to a contant class

File: ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs [hunk 3]
```diff
@@ -15,5 +18,21 @@ public static ProgressDialog CreateDialog(string title, string message, Context
...
             return progressDialog;
         }
+
+        public static void ShowLogoutDialog(Context context)
+        {
+            var preferencesEditor = PreferenceManager.GetDefaultSharedPreferences(Application.Context).Edit();
+            new AlertDialog.Builder(context)
+                .SetCancelable(false)
+                .SetTitle("Are you sure to logout?")
+                .SetPositiveButton("Yes", (dce, e) =>
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Ultilities/DialogExtensions.cs]: Put `Yes` to `Strings.xml`

File: ClubManagement/ClubManagement/Resources/layout/fragment_money.axml [hunk 4]
```diff
@@ -17,6 +17,15 @@
             android:textColor="@color/text_color_white"
             android:textSize="@dimen/tabbar_text_size"
             android:textStyle="bold" />
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:text="Log out"
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Resources/layout/fragment_money.axml]: Put this button on the Dashboard only

File: ClubManagement/ClubManagement/Resources/layout/fragment_dashboard.axml [hunk 5]
```diff
@@ -16,6 +16,15 @@
             android:textColor="@color/text_color_white"
             android:textSize="@dimen/tabbar_text_size"
             android:textStyle="bold" />
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:text="Log out"
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Resources/layout/fragment_dashboard.axml]: Extract to Strings.xml

File: ClubManagement/ClubManagement/Resources/layout/FragmentBalance.axml [hunk 6]
```diff
@@ -17,6 +17,15 @@
             android:textColor="@color/text_color_white"
             android:textSize="@dimen/tabbar_text_size"
             android:textStyle="bold" />
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Resources/layout/FragmentBalance.axml]: Put this button on the Dashboard only

File: ClubManagement/ClubManagement/Fragments/MoneyFragment.cs [hunk 7]
```diff
@@ -1,33 +1,42 @@
...
 
-        private readonly AppDataController appDataController = AppDataController.Instance;
+        [InjectOnClick(Resource.Id.btnLogout)]
+        private void Logout(object s, EventArgs e)
+        {
+            DialogExtensions.ShowLogoutDialog(Context);
+        }
 
-        private List<MoneyState> listMoneyStates;
+        private const string AllTabTitle = "All";
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Fragments/MoneyFragment.cs]: Extract to `Strings.xml`

Comment (nguyenvanman) [reply ClubManagement/ClubManagement/Fragments/MoneyFragment.cs]: all done

**Pull request merged**